css/style.css

```diff
@@ -1,3 +1,7 @@
 body {
   text-align: center;
+}
+
+main {
+  padding: 120px;
```

Comment:
Esse espaço serve pra que? se for pra compensar o header fixo, vale a pena deixa somente no `top`. padding-top

Reply:
__Sim, era para isso mesmo. Já alterei para padding-top e atualizei a descrição do PR incluindo o normalize.css =)